client/my-sites/domains/controller.jsx

```diff
@@ -212,6 +213,25 @@ const googleAppsWithRegistration = ( context, next ) => {
 	next();
 };
 
+const emailUpsellForDomainRegistration = ( context, next ) => {
+	context.primary = (
+		<Main wideLayout>
+			<PageViewTracker
+				path="/domains/add/:domain/email/:site"
+				title="Domain Search > Domain Registration > Email"
+			/>
+			<DocumentHead
+				title={ translate( 'Register %(domain)s', {
+					args: { domain: context.params.registerDomain },
```

Comment:
Small bug: this should be `context.params.domain` - `registerDomain` is not defined in the main URL path as a variable.
```suggestion
					args: { domain: context.params.domain },
```

Reply:
Whoops, gonna fix that.

Reply:
Done.